Libs/Optimize/OptimizeParameters.cpp

```diff
@@ -215,6 +215,12 @@ int OptimizeParameters::get_verbosity() { return this->params_.get("verbosity",
 //---------------------------------------------------------------------------
 void OptimizeParameters::set_verbosity(int value) { this->params_.set("verbosity", value); }
 
+//---------------------------------------------------------------------------
+int OptimizeParameters::get_mesh_ffc_mode() { return this->params_.get("mesh_ffc_mode", 0); }
```

Comment:
I believe this should be a **bool**, not an **int**.  I assume it's either on or off.

Reply:
Yes, how does bool work here? Does it read 0/1 or does it read true/false, or is it True/False like in python? I opted for int cause at least I know 0/1 works, but I agree that bool is better.

Reply:
This is implemented in `Variant::operator bool()` here:

https://github.com/SCIInstitute/ShapeWorks/blob/0ca7bde21d3089b2ec7737154fb7dfd565c73ff3/Libs/Project/Variant.cpp#L16

It should accept 0/1, true/false, True/False, etc.